pkgs/development/interpreters/python/cpython/default.nix

```diff
@@ -161,6 +167,9 @@ let
     pythonOnBuildForHost
   ] ++ optionals (stdenv.cc.isClang && (!stdenv.hostPlatform.useAndroidPrebuilt or false) && (enableLTO || enableOptimizations)) [
     stdenv.cc.cc.libllvm.out
```

Comment:
To adapt for darwin system, this should be modified because it overrides `llvm` at the last of `nativeBuildInputs`. Python 3.13.1 tries to find `llvm-readobj-18` and `libllvm.out` defaults to 16, which is not acceptable.

A possible solution is adding a condition `&& (!jitSupport)`, but this is a little ugly.